draft-ietf-gnap-resource-servers.md

```diff
@@ -402,6 +418,25 @@ repeat this process as necessary for calling further RS's.
 \[\[ TBD: There are a lot of items in the document that are expandable
 through the use of value registries. \]\]
 
+## GNAP Token Format Registry {#iana-token-format}
+
+This specification establishes the GNAP Token Format Registry to define token formats.
+
+### Registry Template
+
+### Initial Registry Contents 
+
+The table below contains the initial contents of the GNAP Token Format Registry.
+
+|Name|Status|Description|Reference|
+|--- |--- |--- |--- |
+|`jwt-signed`|Active   | JSON Web Token, signed with JWS | {{RFC7519}} |
```

Comment:
Please explain what "active" means, and what is the alternative: "deprecated"?

Also, there's other stuff needed when defining a registry, such as allocation policy etc.

Reply:
Agreed, the rest of the registry framework isn't in there, this was a placeholder for the list of initial values. The core document needs this as well.